### ISSUE TITLE
[slyguy.disney.plus] cannot fetch from URL

### ISSUE THREAD

**Issue by stei-f** [matthuisman/slyguy.addons#466]
Hi,

I hope I'm correct here.
**Environment:**
kodi                          6:20.1+git20230312.0705-289ec664e3-0~focal
on Ubuntu 20.04.6 LTS
Addon: Disney+, slyguy - 0.13.8

Following error pulled from **debug log:**

```
2023-06-04 10:45:42.085 T:1987    debug <general>: slyguy.disney.plus - Cache data loaded
2023-06-04 10:45:42.088 T:1987    debug <general>: slyguy.disney.plus - Rewrites Loaded: 1
2023-06-04 10:45:42.089 T:1987    debug <CSettingsManager>: requested setting (proxy_server) was not found.
2023-06-04 10:45:42.090 T:1987    debug <general>: slyguy.disney.plus - Router Parsed: '?_=collection&content_class=contentType&slug=movies' => collection {'content_class': 'contentType', 'slug': 'movies', '_url': 'plugin://slyguy.disney.plus?_=collection&content_class=contentType&slug=movies'}
2023-06-04 10:45:42.090 T:1987    debug <general>: slyguy.disney.plus - Attempt 1/2: GET https://bam-sdk-configs.bamgrid.com/bam-sdk/v4.0/disney-svod-3d9324fc/android/v8.2.4/google/tv/prod.json
2023-06-04 10:45:42.093 T:1987    debug <general>: slyguy.disney.plus - SIGNAL: on_error
2023-06-04 10:45:42.093 T:1987    debug <general>: slyguy.disney.plus - Mem Cache: Deleted 0 Rows
2023-06-04 10:45:42.094 T:1987    debug <general>: slyguy.disney.plus - SIGNAL: on_close
2023-06-04 10:45:42.103 T:1987    debug <general>: slyguy.disney.plus - URL lieferte kein Ergebnis
                                                   Bitte kontrolliere Deine Internetverbindung. Stelle sicher, dass Deine IP Adresse nicht durch Geo-Blocking gesperrt ist.
                                                   Traceback (most recent call last):
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/signals.py", line 41, in throwable
                                                       yield
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/router.py", line 121, in dispatch
                                                       function(**params)
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 62, in decorated_function
                                                       item = f(*args, **kwargs)
                                                     File "/home/kodi/.kodi/addons/slyguy.disney.plus/resources/lib/plugin.py", line 195, in collection
                                                       data = api.collection_by_slug(slug, content_class, 'PersonalizedCollection' if slug == 'home' else 'StandardCollection')
                                                     File "/home/kodi/.kodi/addons/slyguy.disney.plus/resources/lib/api.py", line 359, in collection_by_slug
                                                       endpoint = self._endpoint(self.get_config()['services']['content']['client']['endpoints']['getCollection']['href'], collectionSubType=sub_type, contentClass=content_class, slug=slug)
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/mem_cache.py", line 115, in decorated_function
                                                       value = f(*args, **kwargs)
                                                     File "/home/kodi/.kodi/addons/slyguy.disney.plus/resources/lib/api.py", line 34, in get_config
                                                       return self._session.get(CONFIG_URL).json()
                                                     File "/usr/lib/python3/dist-packages/requests/sessions.py", line 546, in get
                                                       return self.request('GET', url, **kwargs)
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 390, in request
                                                       raise SessionError(error_msg or _.NO_RESPONSE_ERROR)
                                                   slyguy.exceptions.SessionError: URL lieferte kein Ergebnis
                                                   Bitte kontrolliere Deine Internetverbindung. Stelle sicher, dass Deine IP Adresse nicht durch Geo-Blocking gesperrt ist.

```

But the url seems to work fine. Checked on multiple host including the kodi client:
```
user@host:~$ wget https://bam-sdk-configs.bamgrid.com/bam-sdk/v4.0/disney-svod-3d9324fc/android/v8.2.4/google/tv/prod.json
--2023-06-04 10:53:17--  https://bam-sdk-configs.bamgrid.com/bam-sdk/v4.0/disney-svod-3d9324fc/android/v8.2.4/google/tv/prod.json
Auflösen des Hostnamens bam-sdk-configs.bamgrid.com (bam-sdk-configs.bamgrid.com) … 99.84.88.110, 99.84.88.121, 99.84.88.50, ...
Verbindungsaufbau zu bam-sdk-configs.bamgrid.com (bam-sdk-configs.bamgrid.com)|99.84.88.110|:443 … verbunden.
HTTP-Anforderung gesendet, auf Antwort wird gewartet … 200 OK
Länge: 74114 (72K) [application/json]
Wird in »prod.json« gespeichert.

prod.json           100%[===================>]  72,38K  --.-KB/s    in 0,02s

2023-06-04 10:53:18 (4,31 MB/s) - »prod.json« gespeichert [74114/74114]

user@host:~$

```

Any hints where and how to debug further on? Help and feedback appreciated, Thanks.

**Comment by matthuisman** [comment]
Upload full debug log plz

**Comment by stei-f** [comment]
Well, while debugging it seems like I triggered an Update or something, but now the addon is refusing to start:

```
2023-06-04 11:38:22.274 T:3859    error <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'AttributeError'>
                                                   Error Contents: module 'urllib3.util.ssl_' has no attribute '_ssl_wrap_socket_impl'
                                                   Traceback (most recent call last):
                                                     File "/home/kodi/.kodi/addons/slyguy.disney.plus/default.py", line 3, in <module>
                                                       from resources.lib.plugin import plugin
                                                     File "/home/kodi/.kodi/addons/slyguy.disney.plus/resources/lib/plugin.py", line 3, in <module>
                                                       from slyguy import plugin, gui, userdata, signals, inputstream, settings
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 12, in <module>
                                                       from . import router, gui, settings, userdata, inputstream, signals, migrate, bookmarks, mem_cache
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/inputstream.py", line 12, in <module>
                                                       from .session import Session
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 40, in <module>
                                                       orig_ssl_wrap_socket_impl = urllib3.util.ssl_._ssl_wrap_socket_impl
                                                   AttributeError: module 'urllib3.util.ssl_' has no attribute '_ssl_wrap_socket_impl'
                                                   -->End of Python script error report<--

2023-06-04 11:38:22.393 T:3858    error <general>: GetDirectory - Error getting plugin://slyguy.disney.plus/
2023-06-04 11:38:22.395 T:1556    error <general>: CGUIMediaWindow::GetDirectory(plugin://slyguy.disney.plus/) failed
```

I've seen this error described in the other bugreport for slyguy common, for which I'm on 0.65.7.
Anyway, trying to pull a full log...

should be this (debug enabled, capture log, go to video and start addon, which fails):
```
2023-06-04 11:46:59.977 T:1541    debug <general>: Keyboard: scancode: 0x24, sym: 0x0d, unicode: 0x0d, modifier: 0x0
2023-06-04 11:47:00.162 T:1541    debug <general>: Keyboard: scancode: 0x24, sym: 0x0d, unicode: 0x00, modifier: 0x0
2023-06-04 11:47:00.162 T:1541    debug <general>: HandleKey: return (0xf00d) pressed, window 10000, action is Select
2023-06-04 11:47:00.162 T:1541    debug <general>: Activating window ID: 10025
2023-06-04 11:47:00.465 T:1541    debug <general>: ------ Window Deinit (Home.xml) ------
2023-06-04 11:47:00.465 T:1541    debug <general>: FreeVisualisation() done
2023-06-04 11:47:00.490 T:1541    debug <general>: ------ Window Init (MyVideoNav.xml) ------
2023-06-04 11:47:00.490 T:1541     info <general>: Loading skin file: MyVideoNav.xml, load type: KEEP_IN_MEMORY
2023-06-04 11:47:00.537 T:1541    debug <general>: CGUIMediaWindow::GetDirectory ()
2023-06-04 11:47:00.537 T:1541    debug <general>:   ParentPath = []
2023-06-04 11:47:00.539 T:3599    debug <general>: Thread BackgroundLoader start, auto delete: false
2023-06-04 11:47:00.542 T:3599    debug <general>: Thread BackgroundLoader 140155382650624 terminating
2023-06-04 11:47:01.594 T:1541    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-06-04 11:47:01.594 T:1541    debug <general>: HandleKey: down (0xf081) pressed, window 10025, action is Down
2023-06-04 11:47:01.779 T:1541    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-06-04 11:47:02.029 T:1541    debug <general>: Skipped 1 duplicate messages..
2023-06-04 11:47:02.029 T:1541    debug <general>: HandleKey: down (0xf081) pressed, window 10025, action is Down
2023-06-04 11:47:02.129 T:1541    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-06-04 11:47:02.704 T:1541    debug <general>: Keyboard: scancode: 0x24, sym: 0x0d, unicode: 0x0d, modifier: 0x0
2023-06-04 11:47:02.830 T:1541    debug <general>: Keyboard: scancode: 0x24, sym: 0x0d, unicode: 0x00, modifier: 0x0
2023-06-04 11:47:02.830 T:1541    debug <general>: HandleKey: return (0xf00d) pressed, window 10025, action is Select
2023-06-04 11:47:02.831 T:1541    debug <general>: CGUIMediaWindow::GetDirectory (library://video/addons.xml/)
2023-06-04 11:47:02.831 T:1541    debug <general>:   ParentPath = []
2023-06-04 11:47:02.832 T:3600    debug <general>: Thread waiting start, auto delete: false
2023-06-04 11:47:02.833 T:3600    debug <general>: Thread waiting 140155511891712 terminating
2023-06-04 11:47:02.836 T:3601    debug <general>: Thread BackgroundLoader start, auto delete: false
2023-06-04 11:47:02.844 T:3601    debug <general>: Thread BackgroundLoader 140155382650624 terminating
2023-06-04 11:47:02.845 T:1541    error <general>: Repository add-on repository.mediathekviewmatrix uses old schema definition for the repository extension point! This is no longer supported, please update your addon to use <dir> definitions.
2023-06-04 11:47:02.845 T:1541    error <general>: Repository add-on repository.mediathekviewmatrix does not have any directory and won't be able to update/serve addons! Please fix the addon.xml definition
2023-06-04 11:47:03.487 T:1541    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-06-04 11:47:03.487 T:1541    debug <general>: HandleKey: down (0xf081) pressed, window 10025, action is Down
2023-06-04 11:47:03.487 T:3602    debug <general>: Thread JobWorker start, auto delete: true
2023-06-04 11:47:03.490 T:3602    debug <general>: ffmpeg[0x563ad3408de0]: [image2] Custom AVIOContext makes no sense and will be ignored with AVFMT_NOFILE format.
2023-06-04 11:47:03.587 T:1541    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-06-04 11:47:03.696 T:1541    debug <general>: Skipped 1 duplicate messages..
2023-06-04 11:47:03.696 T:1541    debug <general>: HandleKey: down (0xf081) pressed, window 10025, action is Down
2023-06-04 11:47:03.698 T:3602    debug <general>: ffmpeg[0x563ad3408de0]: [image2] Custom AVIOContext makes no sense and will be ignored with AVFMT_NOFILE format.
2023-06-04 11:47:03.845 T:1541    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-06-04 11:47:04.447 T:1541    debug <general>: Skipped 1 duplicate messages..
2023-06-04 11:47:04.447 T:1541    debug <general>: HandleKey: down (0xf081) pressed, window 10025, action is Down
2023-06-04 11:47:04.450 T:3602    debug <general>: ffmpeg[0x563ad3408de0]: [image2] Custom AVIOContext makes no sense and will be ignored with AVFMT_NOFILE format.
2023-06-04 11:47:04.545 T:1541    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-06-04 11:47:04.612 T:1541    debug <general>: Skipped 1 duplicate messages..
2023-06-04 11:47:04.612 T:1541    debug <general>: HandleKey: down (0xf081) pressed, window 10025, action is Down
2023-06-04 11:47:04.615 T:3602    debug <general>: ffmpeg[0x563ad3408de0]: [image2] Custom AVIOContext makes no sense and will be ignored with AVFMT_NOFILE format.
2023-06-04 11:47:04.745 T:1541    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-06-04 11:47:04.812 T:1541    debug <general>: Skipped 1 duplicate messages..
2023-06-04 11:47:04.812 T:1541    debug <general>: HandleKey: down (0xf081) pressed, window 10025, action is Down
2023-06-04 11:47:04.813 T:3602    debug <general>: ffmpeg[0x563ad3408de0]: [image2] Custom AVIOContext makes no sense and will be ignored with AVFMT_NOFILE format.
2023-06-04 11:47:04.929 T:1541    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-06-04 11:47:04.996 T:1541    debug <general>: Skipped 1 duplicate messages..
2023-06-04 11:47:04.996 T:1541    debug <general>: HandleKey: down (0xf081) pressed, window 10025, action is Down
2023-06-04 11:47:04.997 T:3602    debug <general>: ffmpeg[0x563ad3408de0]: [image2] Custom AVIOContext makes no sense and will be ignored with AVFMT_NOFILE format.
2023-06-04 11:47:05.096 T:1541    debug <general>: Keyboard: scancode: 0x74, sym: 0x112, unicode: 0x00, modifier: 0x0
2023-06-04 11:47:06.999 T:1541    debug <general>: Keyboard: scancode: 0x24, sym: 0x0d, unicode: 0x0d, modifier: 0x0
2023-06-04 11:47:07.126 T:1541    debug <general>: Keyboard: scancode: 0x24, sym: 0x0d, unicode: 0x00, modifier: 0x0
2023-06-04 11:47:07.126 T:1541    debug <general>: HandleKey: return (0xf00d) pressed, window 10025, action is Select
2023-06-04 11:47:07.127 T:1541    debug <general>: CGUIMediaWindow::GetDirectory (plugin://slyguy.disney.plus/)
2023-06-04 11:47:07.127 T:1541    debug <general>:   ParentPath = [addons://sources/video]
2023-06-04 11:47:07.128 T:3603    debug <general>: Thread waiting start, auto delete: false
2023-06-04 11:47:07.131 T:3602    debug <general>: CAddonDatabase::SetLastUsed[slyguy.disney.plus] took 3 ms
2023-06-04 11:47:07.131 T:3604    debug <general>: Thread JobWorker start, auto delete: true
2023-06-04 11:47:07.131 T:3603    debug <general>: CScriptRunner: running add-on script Disney+('plugin://slyguy.disney.plus/', '1', '')
2023-06-04 11:47:07.131 T:3605    debug <general>: Thread LanguageInvoker start, auto delete: false
2023-06-04 11:47:07.131 T:3605    debug <general>: initializing python engine.
2023-06-04 11:47:07.132 T:3605    debug <general>: CPythonInvoker(7, /home/kodi/.kodi/addons/slyguy.disney.plus/default.py): start processing
2023-06-04 11:47:07.196 T:3605    debug <general>: -->Python Interpreter Initialized<--
2023-06-04 11:47:07.196 T:3605    debug <general>:

2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7, /home/kodi/.kodi/addons/slyguy.disney.plus/default.py): the source file to load is "/home/kodi/.kodi/addons/slyguy.disney.plus/default.py"
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7): default python path:
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7):   /usr/lib/python38.zip
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7):   /usr/lib/python3.8
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7):   /usr/lib/python3.8/lib-dynload
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7):   /usr/local/lib/python3.8/dist-packages
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7):   /usr/lib/python3/dist-packages
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7): adding path:
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7):  /home/kodi/.kodi/addons/script.module.slyguy/resources/modules
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7):  /home/kodi/.kodi/addons/slyguy.dependencies/resources/modules
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7):  /home/kodi/.kodi/addons/slyguy.disney.plus
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7): adding args:
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7):  plugin://slyguy.disney.plus/
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7):  1
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7):
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7):  resume:false
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7, /home/kodi/.kodi/addons/slyguy.disney.plus/default.py): entering source directory /home/kodi/.kodi/addons/slyguy.disney.plus
2023-06-04 11:47:07.196 T:3605    debug <general>: CPythonInvoker(7, /home/kodi/.kodi/addons/slyguy.disney.plus/default.py): instantiating addon using automatically obtained id of "slyguy.disney.plus" dependent on version 0.0.0 of the xbmc.python api
2023-06-04 11:47:07.228 T:1541    debug <general>: ------ Window Init (DialogBusy.xml) ------
2023-06-04 11:47:07.329 T:3605    debug <general>: slyguy.disney.plus - sys.path: ['/usr/lib/python38.zip', '/usr/lib/python3.8', '/usr/lib/python3.8/lib-dynload', '/usr/local/lib/python3.8/dist-packages', '/usr/lib/python3/dist-packages', '/home/kodi/.kodi/addons/script.module.slyguy/resources/modules', '/home/kodi/.kodi/addons/slyguy.dependencies/resources/modules', '/home/kodi/.kodi/addons/slyguy.disney.plus']
2023-06-04 11:47:07.864 T:3605    debug <general>: slyguy.disney.plus - OpenSSL 1.1.1f  31 Mar 2020
2023-06-04 11:47:07.868 T:3605    error <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'AttributeError'>
                                                   Error Contents: module 'urllib3.util.ssl_' has no attribute '_ssl_wrap_socket_impl'
                                                   Traceback (most recent call last):
                                                     File "/home/kodi/.kodi/addons/slyguy.disney.plus/default.py", line 3, in <module>
                                                       from resources.lib.plugin import plugin
                                                     File "/home/kodi/.kodi/addons/slyguy.disney.plus/resources/lib/plugin.py", line 3, in <module>
                                                       from slyguy import plugin, gui, userdata, signals, inputstream, settings
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 12, in <module>
                                                       from . import router, gui, settings, userdata, inputstream, signals, migrate, bookmarks, mem_cache
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/inputstream.py", line 12, in <module>
                                                       from .session import Session
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 40, in <module>
                                                       orig_ssl_wrap_socket_impl = urllib3.util.ssl_._ssl_wrap_socket_impl
                                                   AttributeError: module 'urllib3.util.ssl_' has no attribute '_ssl_wrap_socket_impl'
                                                   -->End of Python script error report<--

2023-06-04 11:47:07.868 T:3605    debug <general>: onExecutionDone(7, /home/kodi/.kodi/addons/slyguy.disney.plus/default.py)
2023-06-04 11:47:07.880 T:1541    debug <general>: ------ Window Init (DialogNotification.xml) ------
2023-06-04 11:47:07.899 T:3605    debug <general>: .
2023-06-04 11:47:07.969 T:3605    debug <general>: Skipped 1 duplicate messages..
2023-06-04 11:47:07.969 T:3605    debug <general>: Python interpreter stopped
2023-06-04 11:47:07.969 T:3605    debug <general>: Thread LanguageInvoker 140153867802368 terminating
2023-06-04 11:47:07.975 T:3603    error <general>: GetDirectory - Error getting plugin://slyguy.disney.plus/
2023-06-04 11:47:07.975 T:3603    debug <general>: Thread waiting 140155493103360 terminating
2023-06-04 11:47:07.979 T:1541    debug <general>: ------ Window Deinit (DialogBusy.xml) ------
2023-06-04 11:47:07.980 T:1541    error <general>: CGUIMediaWindow::GetDirectory(plugin://slyguy.disney.plus/) failed
2023-06-04 11:47:07.980 T:1541    debug <general>: CGUIMediaWindow::GetDirectory (addons://sources/video)
2023-06-04 11:47:07.980 T:1541    debug <general>:   ParentPath = []
2023-06-04 11:47:07.980 T:3607    debug <general>: Thread waiting start, auto delete: false
2023-06-04 11:47:07.980 T:3607    debug <general>: Thread waiting 140155493103360 terminating
2023-06-04 11:47:07.982 T:3608    debug <general>: Thread BackgroundLoader start, auto delete: false
2023-06-04 11:47:07.986 T:3608    debug <general>: Thread BackgroundLoader 140155382650624 terminating
2023-06-04 11:47:13.784 T:1541    debug <general>: ------ Window Deinit (DialogNotification.xml) ------

```

Sorry for the confusion, but I still do not know why all of a sudden the error did change...

**Comment by stei-f** [comment]
OK I think I did trigger an update of script.module.slyguy:

here I grepped in kodi.old.log:

```
#> grep script.module.slyguy kodi.old.log
2023-06-04 11:02:58.294 T:1630    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
2023-06-04 11:02:58.298 T:1630    debug <general>: script.module.slyguy - Rewrites Loaded: 1
2023-06-04 11:02:58.299 T:1630    debug <general>: script.module.slyguy - Attempt 1/2: GET https://slyguy.uk/.repo/news.json.gz
2023-06-04 11:02:58.300 T:1630    error <general>: script.module.slyguy - URL lieferte kein Ergebnis
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/lib/service.py", line 103, in start
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/lib/service.py", line 28, in _check_news
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 350, in gz_json
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 390, in request
2023-06-04 11:02:58.401 T:1630    debug <general>: script.module.slyguy - SKIPPED SIGNAL: on_settings_changed
2023-06-04 11:10:42.305 T:2707    debug <general>: CPythonInvoker(8):  /home/kodi/.kodi/addons/script.module.slyguy/resources/modules
2023-06-04 11:10:43.011 T:2707    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
2023-06-04 11:10:46.401 T:2707    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/signals.py", line 41, in throwable
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/router.py", line 121, in dispatch
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 62, in decorated_function
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/mem_cache.py", line 115, in decorated_function
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 390, in request
2023-06-04 11:10:48.642 T:2707    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
2023-06-04 11:11:37.824 T:2707    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/signals.py", line 41, in throwable
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/router.py", line 121, in dispatch
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 62, in decorated_function
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/mem_cache.py", line 115, in decorated_function
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 390, in request
2023-06-04 11:16:31.183 T:2970    debug <general>: CPythonInvoker(10):  /home/kodi/.kodi/addons/script.module.slyguy/resources/modules
2023-06-04 11:16:31.896 T:2970    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
2023-06-04 11:16:34.395 T:2970    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
2023-06-04 11:16:54.609 T:2970    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
2023-06-04 11:16:54.649 T:2970    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
2023-06-04 11:16:57.169 T:2970    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/signals.py", line 41, in throwable
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/router.py", line 121, in dispatch
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 62, in decorated_function
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/mem_cache.py", line 115, in decorated_function
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 390, in request
2023-06-04 11:16:59.028 T:2970    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
2023-06-04 11:17:19.934 T:2970    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
2023-06-04 11:17:23.820 T:2970    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/signals.py", line 41, in throwable
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/router.py", line 121, in dispatch
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 62, in decorated_function
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/mem_cache.py", line 115, in decorated_function
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 390, in request
2023-06-04 11:17:25.127 T:2970    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
2023-06-04 11:17:26.105 T:2967    debug <general>: ADDONS: update check: addonID = script.module.slyguy / Origin = repository.slyguy / Version = 0.65.4
2023-06-04 11:17:26.145 T:2967    debug <general>: ADDONS: update check: addonID = script.module.slyguy / Origin = repository.slyguy / Version = 0.65.4
2023-06-04 11:17:28.326 T:2970    debug <CAddonSettings[0@script.module.slyguy]>: trying to load setting definitions from old format...
2023-06-04 11:18:05.309 T:3046    debug <general>: ADDONS: update check: addonID = script.module.slyguy / Origin = repository.slyguy / Version = 0.65.4
2023-06-04 11:18:05.356 T:3046    debug <general>: ADDONS: update check: addonID = script.module.slyguy / Origin = repository.slyguy / Version = 0.65.4
2023-06-04 11:21:56.753 T:3072     info <general>: CAddonMgr::FindAddon: script.module.slyguy v0.65.7 installed
2023-06-04 11:21:58.295 T:3194     info <general>: script.module.slyguy - Python Version: 3.8.10 (default, Mar 13 2023, 10:26:41)
2023-06-04 11:21:58.298 T:3194     info <general>: script.module.slyguy - Proxy Started: http://127.0.0.1:52103/
2023-06-04 11:21:58.299 T:3194     info <general>: script.module.slyguy - Widevine Level (fallback): 3
2023-06-04 11:21:58.299 T:3194     info <general>: script.module.slyguy - HDCP Level (fallback): 0.0
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 12, in <module>
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/inputstream.py", line 12, in <module>
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 40, in <module>
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 12, in <module>
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/inputstream.py", line 12, in <module>
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 40, in <module>
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 12, in <module>
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/inputstream.py", line 12, in <module>
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 40, in <module>
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/plugin.py", line 12, in <module>
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/inputstream.py", line 12, in <module>
                                                     File "/home/kodi/.kodi/addons/script.module.slyguy/resources/modules/slyguy/session.py", line 40, in <module>
2023-06-04 11:33:46.999 T:3654     info <general>: CAddonMgr::FindAddon: script.module.slyguy v0.65.7 installed
2023-06-04 11:33:47.810 T:3695     info <general>: script.module.slyguy - Python Version: 3.8.10 (default, Mar 13 2023, 10:26:41)
2023-06-04 11:33:47.813 T:3695     info <general>: script.module.slyguy - Proxy Started: http://127.0.0.1:52103/
2023-06-04 11:33:47.814 T:3695     info <general>: script.module.slyguy - Widevine Level (fallback): 3
2023-06-04 11:33:47.814 T:3695     info <general>: script.module.slyguy - HDCP Level (fallback): 0.0
```
What I see is, I had the URL-load-issue, Than there was an update of the module. And then the output of the module changed to also include the "info" sections.

**Comment by stei-f** [comment]
ok, this is probably not helpfull to the developer, but revert script.module.slyguy to 0.65.1 fixed everything for me.

Let me know when I can test something for you, like installing debug build or alike.

**Comment by matthuisman** [comment]
fixed with slyguy common 0.65.8
